### PR TITLE
Add displaysize(), fix version script

### PR DESCRIPTION
**Pull request by nalimilan**
Two not-so-related fixes.

### CODE REVIEW

**Comment by nalimilan** [comment]
Regarding the first commit, it would be much more useful if we defined ```displaysize(io::IO) = tty_size()`, which would allow people to use the correct API on 0.5, and fallback to the REPL-limited one on 0.4. But it might also be confusing to ignore completely `io`. Opinions?

File: src/Compat.jl [hunk 1]
```diff
@@ -761,7 +761,7 @@ if VERSION < v"0.5.0-dev+1946"
     export supertype
 end
 
-if VERSION < v"0.4.0-dev+1653"
+if VERSION < v"0.5.0-dev+1653"
```
Comment (tkelman) [inline src/Compat.jl]: which PR was this in base?

Comment (nalimilan) [inline src/Compat.jl]: Ah, nevermind, this was apparently correct. I looked for `rm_recursive` in 0.4 to check whether it was already present, but I should have checked `mktemp` and `mktempdir` instead.

**Comment by tkelman** [comment]
test, readme

**Comment by tkelman** [comment]
Is it possible to test `displaysize`? Needs a rebase.

**Comment by nalimilan** [comment]
It doesn't look like a proper test can be done easily (I don't see one in Base), but at least I've added a test that the function returns a tuple of two ints as expected. Anyway, the correctness of the result itself should be checked by base; the only failure which could happen in Compat is that the function wouldn't be defined correctly.

**Comment by tkelman** [comment]
Makes sense.

**Pull request merged**